Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -625,6 +625,29 @@ class PaymentSheetStandardLPMUITests: PaymentSheetUITestCase {
         payButton.tap()
     }
 
+    func testSunbitPaymentMethod() throws {
+        var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
+        settings.currency = .usd
+        settings.merchantCountryCode = .US
+        settings.customerMode = .new
+        settings.apmsEnabled = .off
+        loadPlayground(app, settings)
+        app.buttons["Present PaymentSheet"].tap()
+        let payButton = app.buttons["Pay $50.99"]
+
+        // Select Sunbit
+        guard let sunbit = scroll(collectionView: app.collectionViews.firstMatch, toFindCellWithId: "Sunbit") else {
+            XCTFail()
+            return
+        }
+        sunbit.tap()
+
+        XCTAssertTrue(payButton.isEnabled)
+
+        // Attempt payment, should succeed
+        payButton.tap()
+    }
+
```

Comment:
I understand you're just copying the existing test pattern here, but I think these tests need to complete payment or they're not testing enough to be useful. You can leave it as-is in this PR, I can't find an example to copy that completes the payment. 

cc @davidme-stripe I think we should make all of these complete payment, there's nothing else that can test this final part of the checkout flow of 'next action handled -> paymenthandler completes payment'.

Reply:
Oof yeah, I don't know where this snuck in. We be testing the entire flow where we can

Reply:
I'll look into extending the test to support this. I'd like this PR to be an example we can reference for future integrations

Reply:
Since the authorization happens using the in-app browser, I wasn't able to find a way to tap the authorize button.

I did add an assertion that validates the in-app browser is open at least...